src/main/java/seedu/address/model/issue/Status.java

```diff
@@ -0,0 +1,8 @@
+package seedu.address.model.issue;
+
+/**
+ * Represents the various completion statuses of an issue.
+ */
+public enum Status {
+    COMPLETED, UNCOMPLETED
```

Comment:
Perhaps boolean might be more appropriate here instead of enum (since there are only two values), some setters and getters on the boolean can then be created